backend/conf/application.conf

```diff
@@ -12,4 +12,8 @@ play.filters.cors {
   allowedHttpMethods = null
   allowedHttpHeaders = null
   preflightMaxAge = 3 days
+}
+
+pfs.servers {
```

Comment:
General comment (applicable for later):

we should use a "application.conf.sample" and add "application.conf" in our gitignore. This will allow us to have a shared base of the application configuration without publicly expose data such as domain name, port, keys...